docs/production-install.rst

```diff
@@ -48,37 +45,32 @@ Then the repository definition:
   enabled=1
   EOF
 
+Install the Socorro repository.
+::
+  sudo rpm -ivh https://s3-us-west-2.amazonaws.com/org.mozilla.crash-stats.packages-public/el/7/noarch/socorro-public-repo-1-1.el7.centos.noarch.rpm
+
 Now you can actually install the packages:
 ::
   sudo yum install postgresql93-server postgresql93-plperl \
     postgresql93-contrib java-1.7.0-openjdk python-virtualenv \
-    rabbitmq-server elasticsearch httpd mod_wsgi memcached daemonize
-
-Enable Apache on startup:
-::
-  sudo service httpd start
-  sudo chkconfig httpd on
+    rabbitmq-server elasticsearch httpd mod_wsgi memcached socorro
 
 Enable Memcached on startup:
 ::
-  sudo service memcached start
-  sudo chkconfig memcached on
+  sudo systemctl enable memcached
 
 Enable RabbitMQ on startup:
 ::
-  sudo service rabbitmq-server start
-  sudo chkconfig rabbitmq-server on
+  sudo systemctl enable rabbitmq-server
 
-Enable RabbitMQ on startup:
+Enable ElasticSearch on startup:
```

Comment:
We write the name of this software in different ways all over the place, heh.  Officially it's "Elasticsearch" (one word, first letter capitalised only).